app/scenes/Settings/components/WebhookSubscriptionForm.tsx

```diff
@@ -250,6 +252,13 @@ function WebhookSubscriptionForm({ handleSubmit, webhookSubscription }: Props) {
           label={t("URL")}
           {...register("url", { required: true })}
         />
+        <ReactHookWrappedInput
+          flex
+          label={t("Secret")}
+          {...register("secret", {
+            required: false,
+          })}
+        />
```

Comment:
Should it be `type="password"` with a button to show/hide?

Reply:
I don't think it's neccessary at the moment